stanford_paragraph_types.module

```diff
@@ -1,7 +1,209 @@
 <?php
+
 /**
  * @file
  * Code for the Stanford Paragraph Types feature.
  */
 
 include_once 'stanford_paragraph_types.features.inc';
+
+/**
+ * Implements hook_theme_registry_alter().
+ *
+ * @see https://www.drupal.org/docs/7/creating-custom-modules/howtos/using-template-tplphp-files-in-your-own-module
+ */
+function stanford_paragraph_types_theme_registry_alter(&$theme_registry) {
+  $module_path = drupal_get_path('module', 'stanford_paragraph_types');
+  $template_objects = drupal_find_theme_templates($theme_registry, '.tpl.php', $module_path);
+  foreach ($template_objects as $key => $template_object) {
+    if (!isset($theme_registry[$key]['theme path']) || !preg_match('#/themes/#', $theme_registry[$key]['theme path'])) {
+      $theme_registry[$key]['theme path'] = $module_path;
+      $theme_registry[$key] = array_merge($theme_registry[$key], $template_object);
+      $theme_registry[$key]['type'] = 'module';
+    }
+  }
+}
+
+/**
+ * Implements hook_preprocess_entity().
+ */
+function stanford_paragraph_types_preprocess_entity(&$vars) {
+  if (isset($vars['paragraphs_item'])) {
+    drupal_add_css(drupal_get_path('module', 'stanford_paragraph_types') . '/css/stanford_paragraph_types.css');
```

Comment:
Ideally the css for each paragraph would be separated out. I would like to see each paragraph as it's own component and everything it needs is loaded only when it is used. Template, css, etc.

Leave it as is for now but this is a future improvement.